### DISCUSSION TITLE
How to train a parallel model?

### DISCUSSION THREAD

**Discussion by Zhou-jiecheng** [MDIL-SNU/SevenNet#65] [Q&A] [answered]
I want to train a model to do scalable MD simulation, how to do this?

**Comment by YutackPark** [comment, marked as answer]
You don't need another training steps to run parallel, scalable MD simulations.

1. Deploy your model via `sevenn_get_model -p {path to checkpoint}`

2. Run LAMMPS with `mpirun` after writing appropriate input script. You can find the guideline from here: https://github.com/MDIL-SNU/SevenNet

Check 'for parallel model' section.

Note that currently parallel MD run can not compute virial pressure. Plus, cuda-aware openMPI is essential for optimal performance. You should choose number of the mpi process equals the number of GPUs you want to use.
